assets/script.js

```diff
@@ -74,3 +74,13 @@ _gaq.push(['_trackPageview']);
   ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
   var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
 })();
+
+//github api requests
+$(document).ready( function() {
```

Comment:
This can go inside the `$(document).ready( function() {` on the top of the file, no need to attach a second event handler.